bots/pr/src/main/java/org/openjdk/skara/bots/pr/PullRequestBot.java

```diff
@@ -182,6 +182,7 @@ public List<WorkItem> getPeriodicItems() {
                 var targetRef = pr.targetRef();
                 var prId = pr.id();
                 if (pr.isOpen()) {
+                    targetRefPRMap.keySet().forEach(key -> targetRefPRMap.get(key).remove(prId));
```

Comment:
I think this could be simplified by just iterating over `targetRefPRMap.values()`.

Reply:
Yes, it should be better.